cms/djangoapps/contentstore/tasks.py

```diff
@@ -370,10 +375,13 @@ def generate_name(cls, arguments_dict):
 
 
 @task(base=CourseImportTask, bind=True)
+# Note: The decorator @set_code_owner_attribute could not be used because
+#   the implementation of this task will break with any additional decorators.
 def import_olx(self, user_id, course_key_string, archive_path, archive_name, language):
     """
     Import a course or library from a provided OLX .tar.gz archive.
     """
+    set_code_owner_attribute_from_module(__name__)
```

Comment:
@timmc-edx: Here is my potential fix.  Luckily we had test coverage on this one.  If any other tasks have a similar issue, but no test coverage, we might not detect the issue until Production.  I'll just have to keep an eye on worker errors when this rolls out.